automated-comments/rust/reverse-string/solution_function_not_found.md

```diff
@@ -0,0 +1,6 @@
+The test suite for the `reverse-string` exercise expects the following function defined in the `src/lib.rs` file:
```

Comment:
```suggestion
The test suite expects the following function defined in the `src/lib.rs` file:
```

Maybe mentioning the exercise name is not needed?

Reply:
I agree that the exercise name is implied via context.  I would commit the suggestion, but no rights to do so.

Reply:
How about rewording it to simply `The test suite for this exercise`?

Reply:
Sounds good!

Reply:
Yeah, that's fine.